bindings/cs/MongoDB.Crypt/MongoDB.Crypt.csproj

```diff
@@ -18,17 +18,17 @@
     <Description>Official MongoDB supported driver for MongoDB. See http://www.mongodb.org/display/DOCS/CSharp+Language+Center for more details.</Description>
     <Authors>markbenvenuto, vincentkam</Authors>
     <PackageIconUrl>http://jobs.mongodb.org/files/logos/889002/889002.png</PackageIconUrl>
-    <!-- <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance> -->
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <PackageDescription>Official .NET Cryptography support library for MongoDB.</PackageDescription>
     <PackageProjectUrl>http://www.mongodb.org/display/DOCS/CSharp+Language+Center</PackageProjectUrl>
-    <!-- <PackageLicenseFile>License.txt</PackageLicenseFile> -->
+    <PackageLicenseFile>License.txt</PackageLicenseFile>
     <PackageTags>mongodb;mongo;nosql</PackageTags>
     <PackageLanguage>en-US</PackageLanguage>
     <IncludeSymbols>true</IncludeSymbols>
   </PropertyGroup>
 
   <PropertyGroup>
-    <MDBDeveloperDirectory>../../../build/$(Configuration)</MDBDeveloperDirectory>
+    <MDBDeveloperDirectory>../../../cmake-build/$(Configuration)</MDBDeveloperDirectory>
```

Comment:
I don't know what this property is.

Reply:
This property specifies the location of where we can find the binaries for `libmongocrypt`. We could probably update the name for clarity if you'd like!

Reply:
Yes please. Perhaps something like `LibMongoCryptBinaries` then?

Reply:
Done.